tests/test_missing.py

```diff
@@ -30,31 +30,37 @@ def _doit(testfile, **kwargs):
     result2 = active[0:2, 4:6, 7:9]
     np.testing.assert_array_equal(mean_result, result2["sum"]/result2["n"])
 
+@pytest.mark.xfail(USE_S3, reason="Missing data not supported in S3 yet")
 def test_partially_missing_data(tmp_path):
     testfile = str(tmp_path / 'test_partially_missing_data.nc')
     r = dd.make_partially_missing_ncdata(testfile)
     _doit(testfile, missing_value=-999.)
 
+@pytest.mark.xfail(USE_S3, reason="Missing data not supported in S3 yet")
 def test_missing(tmp_path):
     testfile = str(tmp_path / 'test_missing.nc')
     r = dd.make_partially_missing_ncdata(testfile)
     _doit(testfile, missing_value=-999.)
 
+# FIXME: why does it pass for S3?
```

Comment:
This makes me wonder if these tests are working correctly without S3.

Reply:
They pass in normal pytest mode, like they used to, because they run with local files and with the POSIX reduce_chunk, Mark. I don't quite understand your bewilderment here

Reply:
My point is that because S3 does not support missing data yet, these tests should all fail when using S3. But only the first two fail.